omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -48,7 +64,7 @@
                 mkdir "/etc/datadog-agent/conf.d/#{dir_name}" unless File.exists? "/etc/datadog-agent/conf.d/#{dir_name}"
                 move "#{install_dir}/bin/agent/dist/conf.d/#{dir_name}/*.yaml*", "/etc/datadog-agent/conf.d/#{dir_name}/", :force=>true
             end
-            move "#{install_dir}/agent/checks.d", "#{install_dir}/checks.d"
```

Comment:
makes sense to remove this line from this software definition, but you have to update the path here: https://github.com/DataDog/datadog-agent/blob/6.0.0-beta.5/omnibus/config/software/datadog-agent-integrations.rb#L28

Reply:
that was supposed to be copy/paste, not cut/paste.... fixing.